modules/AOW_Actions/actions/actionComputeField.php

```diff
@@ -48,32 +48,34 @@
  */
 class actionComputeField extends actionBase
 {
-
+    
     const RAW_VALUE = "raw";
     const FORMATTED_VALUE = "formatted";
-
+    
     /**
      * @return array
      */
     public function loadJS()
     {
         return array('modules/AOW_Actions/actions/actionComputeField.js');
     }
-
+    
     /**
      * @param SugarBean $bean
-     * @param array $params
-     * @param bool $in_save
+     * @param array     $params
+     * @param bool      $in_save
      *
      * @return bool|void
      */
     public function run_action(SugarBean $bean, $params = array(), $in_save = false)
     {
-        try {
-            require_once('modules/AOW_Actions/FormulaCalculator.php');
-
+        try
```

Comment:
More PSR 2 issues here

Reply:
[SuiteCRM-Coding-Standard.zip](https://github.com/salesagility/SuiteCRM/files/1527675/SuiteCRM-Coding-Standard.zip)

Here are the xml files we use.

Reply:
WOW, thanks!

Reply:
Can we remove the namespaces and autoloading for now?

We can just stick to a straight require_once, it would make it easier to merge your pull request. I think we should look into PSR-0 and PSR-4 for the the modules directory. However, the base class you need to extend doesn't belong to a namespace. We need to be careful about how to structure the namespaces.